src/index.ts

```diff
@@ -13,6 +14,8 @@ app.use(express.json());
 
 app.use("/api", apiRoutes);
 
+app.use(invalidPathHandler);
```

Comment:
it's not a "middleware" and it's not a "path" + what code should be used for not found?

Reply:
https://stackoverflow.com/questions/6528876/how-to-redirect-404-errors-to-a-page-in-expressjs
```
app.use(function(req, res, next) {
  res.status(404);

  if (req.accepts('json')) {
    res.json({ error: 'Not found' });
    return;
  }

  res.type('txt').send('Not found');
});
```

Reply:
done